packages/gatsby-plugin-less/src/gatsby-node.js

```diff
@@ -13,6 +13,7 @@ exports.onCreateWebpackConfig = (
     options: {
       sourceMap: !PRODUCTION,
       ...lessOptions,
```

Comment:
Note it's important to put this AFTER `...lessOptions` because the lessOptions always contain a plugin attribute which would override the  `lessPlugins` attribute

Reply:
This makes sense. Could you update plugin's README to show that to use less plugins, the `lessPlugins` option need to be used?